contracts/cw-proposal-single/src/contract.rs

```diff
@@ -715,14 +727,16 @@ pub fn migrate(_deps: DepsMut, _env: Env, _msg: MigrateMsg) -> Result<Response,
 
 #[cfg_attr(not(feature = "library"), entry_point)]
 pub fn reply(deps: DepsMut, _env: Env, msg: Reply) -> Result<Response, ContractError> {
-    if msg.id % 2 == 0 {
+    if msg.id == FAILED_PROPOSAL_REPLY_ID {
+        Ok(Response::new())
+    } else if msg.id % 2 == 0 {
         // Proposal hook so we can just divide by two for index
-        let idx = msg.id / 2;
+        let idx = msg.id / 2 - 1;
         PROPOSAL_HOOKS.remove_hook_by_index(deps.storage, idx)?;
         Ok(Response::new())
     } else {
         // Vote hook so we can minus one then divid by two for index
-        let idx = (msg.id - 1) / 2;
+        let idx = (msg.id - 1) / 2 - 1;
```

Comment:
Hey, no substracting, since I changed index to start at 1 here: https://github.com/DA0-DA0/dao-contracts/pull/385/files#diff-a9c3c187cde5e8b406a240b2191e7c0e0cb51d1a52b4e3f78e37a23c6c24a481L38-R39
But you are right, it is redundant, since I only needed to reserve `0`, not `0` and `1`. Thanks for noticing it!